src/main/java/com/tabisketch/controller/IndexController.java

```diff
@@ -0,0 +1,12 @@
+package com.tabisketch.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@RequestMapping("/")
+public class IndexController {
+    @GetMapping()
+    public String index() {
+        return "index";
+    }
```

Comment:
top が index になるから、このルートも欲しい！
```suggestion
    }
    
    @GetMapping("/top")
    public String top() {
        return "index";
    }
```

Reply:
修正しました！
https://github.com/saitokazumasa/tabisketch/pull/52/commits/dce81b5350fd0f45713f0837758718e61cb341f2

Reply:
OK!